math/src/polynomial/multilinear_term.rs

```diff
@@ -0,0 +1,168 @@
+use crate::field::element::FieldElement;
+use crate::field::traits::{IsField, IsPrimeField};
+use crate::polynomial::term::Term;
+
+/// Struct for (coeff: FieldElement<F>, terms: Vec<usize>) representing a multilinear
+/// monomial in a sparse format.
+// TODO: add check that var labels are 0 indexed
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct MultiLinearMonomial<F: IsField + IsPrimeField>
+where
+    <F as IsField>::BaseType: Send + Sync,
+{
+    pub coeff: FieldElement<F>,
+    pub vars: Vec<usize>,
+}
+
+impl<F: IsField + IsPrimeField> MultiLinearMonomial<F>
+where
+    <F as IsField>::BaseType: Send + Sync,
+{
+    /// Create a new `Term` from a tuple of the form `(coeff, (variables))`
```

Comment:
Can we have an example of what a Term is here ? It's not really clear what the abstraction is